### PR TITLE
add-splash-time

### PR DESCRIPTION
**Pull request by WinnieXU1**
1.add function in model qemu: splash time before booting, menu on/off, splash picture
2.add an infrasim logo as splash picture
3.add a test for splash time
4.modify the boot-order test with timeout to save test time

### CODE REVIEW

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_boot_splash_time 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
[ 9754   ] test-socat starts to run
[ 9757   ] test-bmc starts to run
[ 9766   ] test-node is running

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
root: INFO: Creating drive: 
root: INFO: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
root: INFO: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/quanta_d51.emu -n -s /home/infrasim/.infrasim/test/data
--------------------- >> end captured logging << --------------------- 
```

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_boot_splash_time 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
[ 15865  ] test-socat starts to run
[ 15868  ] test-bmc starts to run
[ 15877  ] test-node is running

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
root: INFO: Creating drive: 
root: INFO: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
root: INFO: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/quanta_d51.emu -n -s /home/infrasim/.infrasim/test/data
--------------------- >> end captured logging << --------------------- 
```

**Comment by WinnieXU1** [comment]
@fub2 @xiar @JuneZhou19 @XiaowenJiang @turtle-fly @chenge3 
hi all, please help me to review :)

**Comment by XiaowenJiang** [comment]
retest this please

File: infrasim/model.py [hunk 1]
```diff
@@ -1523,9 +1534,17 @@ def handle_parms(self):
                     self.__boot_order = "d"
                 else:
                     self.__boot_order = "ncd"
-                self.add_option("-boot {}".format(self.__boot_order))
+                boot_param.append("{}".format(self.__boot_order))
             else:
-                self.add_option("-boot {}".format(self.__boot_order))
+                boot_param += self.__boot_order
+        if self.__boot_menu:
+            boot_param.append(",menu={}".format(self.__boot_menu))
```
Comment (turtle-fly) [inline infrasim/model.py]: boot_param.append("menu={}".format(...)), without the leading comma, then use `",".join` in line 1547

**Pull request merged**